### DISCUSSION TITLE
Datalist_file link broken in tutorial notebook auto_runner.ipynb

### DISCUSSION THREAD

**Discussion by alessando-guida** [Project-MONAI/tutorials#1561] [Q&A] [answered]
**Describe the bug**

bug found in
[https://github.com/Project-MONAI/tutorials/blob/main/auto3dseg/notebooks/auto_runner.ipynb](https://github.com/Project-MONAI/tutorials/blob/main/auto3dseg/notebooks/auto_runner.ipynb)

After the Hippocampus dataset is downloaded, the ```datalist_file``` is defined as:

`datalist_file = os.path.join("..", "tasks", "msd", msd_task, "msd_" + msd_task.lower() + "_folds.json")
`

This is supposed to be located here:

'../tasks/msd/Task04_Hippocampus/msd_task04_hippocampus_folds.json'

however the file is never created and the path with the dataset paths is not dumpt in there as in other example notebooks. As a consequence, autorunner fails to run and generates the following error:

```
ValueError: Datalist file is not found ../tasks/msd/Task04_Hippocampus/msd_task04_hippocampus_folds.json

```

**Comment by KumoLiu** [comment, marked as answer]
Hi @alessando-guida, did you clone the whole repo? I think the "datalist_file" is located [here](https://github.com/Project-MONAI/tutorials/tree/main/auto3dseg/tasks/msd/Task04_Hippocampus) instead of automatically downloading.

Hope it helps, thanks!

**Reply by alessando-guida** [reply]
Thanks for pointing me to the correct file. I had not cloned the entire repository, I was just trying to use the notebook. I ended up  trying to use the `dataset.json` file that was downloaded with the dataset (Task04_Hippocampus) but I will now try with this new file.
